hpi-swt2/lecture-portal
# Feature/#196 automatic lecture (de) activation

# Relates to #196 

This PR adds the feature of automatic lecture starting/ending.

Acceptance Criteria:
- [x] The lecture states follow the behavior specified in the diagram (https://drive.google.com/file/d/164c58rieeQvvqHmFXsPfiEOPw1B6m8Ig/view?usp=sharing) (**blue parts**)
  - “Running” means:  Same as “Active”, (see here: #197 ) but the lecture is also listed in the currently running lecture list. 
- [x] The lecture control buttons for starting and ending a lecture no longer exist.
- [x] Comprehension traffic light only accepts input while the lecture is running
- [x] Start and end lecture buttons are removed from the UI 


Testing steps:
- create lectures and have fun with the automatic transitions
- see different states here: https://drive.google.com/file/d/164c58rieeQvvqHmFXsPfiEOPw1B6m8Ig/view?usp=sharing

Steps needed for migration:
- none, but you could think about re-applying seeds to db

---

Definition Of Done:
